app/gui2/src/template.vue

```diff
@@ -1,6 +1,6 @@
 <script setup lang="ts">
-const props = defineProps<{}>()
-const emit = defineEmits<{}>()
+const _props = defineProps<{}>()
```

Comment:
not sure this is a good idea - especially considering they *are* meant to be used, it's just that this is a template so there is no elements (and no code) that use them

Reply:
Yes, I guess I could add an eslint ignore on that file. Tbh we could probably move that template to a snippet in project's vscode config.

Reply:
Ended up removing that file and adding a vscode snippet instead.